lib/request-wrapper.ts

```diff
@@ -240,6 +260,7 @@ export class RequestWrapper {
       headers,
       params: qs,
       data,
+      raxConfig: this.raxConfig,
```

Comment:
So I don't actually think this needs to be conditional - if `raxConfig` is not set, this will just be undefined and it shouldn't do anything

Reply:
Great, I'll use this approach and pass undefined when it's undefined